plugins/GSdx/Renderers/Common/GSDevice.h

```diff
@@ -116,6 +122,78 @@ class ShadeBoostConstantBuffer
 
 class GSDevice : public GSAlignedClass<32>
 {
+public:
+	// The order of the values must stay the same for the OGL/DX11 conversion functions to work correctly
+	enum : uint16
+	{
+		// HW blend factors
+		SRC_COLOR,   INV_SRC_COLOR,    DST_COLOR,  INV_DST_COLOR,
+		SRC1_COLOR,  INV_SRC1_COLOR,   SRC_ALPHA,  INV_SRC_ALPHA,
+		DST_ALPHA,   INV_DST_ALPHA,    SRC1_ALPHA, INV_SRC1_ALPHA,
+		CONST_COLOR, INV_CONST_COLOR,  ONE,        ZERO,
+
+		// HW blend operations
+		ADD, SUBTRACT, REV_SUBTRACT
+	};
+
+	static uint16 ConvertBlendEnumOGL(uint16 hw)
+	{
+		static uint16 map[] = 
+		{
+			GL_SRC_COLOR,      GL_ONE_MINUS_SRC_COLOR,      GL_DST_COLOR,  GL_ONE_MINUS_DST_COLOR,
+			GL_SRC1_COLOR,     GL_ONE_MINUS_SRC1_COLOR,     GL_SRC_ALPHA,  GL_ONE_MINUS_SRC_ALPHA,
+			GL_DST_ALPHA,      GL_ONE_MINUS_DST_ALPHA,      GL_SRC1_ALPHA, GL_ONE_MINUS_SRC1_ALPHA,
+			GL_CONSTANT_COLOR, GL_ONE_MINUS_CONSTANT_COLOR, GL_ONE,        GL_ZERO,
+
+			GL_FUNC_ADD, GL_FUNC_SUBTRACT, GL_FUNC_REVERSE_SUBTRACT
+		};
+		return map[hw];
+	}
+
+	static uint16 ConvertBlendEnumDX11(uint16 hw)
```

Comment:
I think you will need to add #ifdef _WIN32 for dx code.

Reply:
Oops, forgot not everyone is a Windows user =P, fixing...